packages/cli/test/simulation/multi_fork.test.ts

```diff
@@ -105,11 +107,41 @@ await checkpointSync.jobs.el.start();
 await checkpointSync.jobs.cl.start();
 await connectNewNode(checkpointSync.nodePair, env.nodes);
 
-await env.waitForNodeSync(rangeSync.nodePair);
-await env.waitForNodeSync(checkpointSync.nodePair);
-
+await waitForNodeSync(rangeSync.nodePair, env.options.controller.signal);
+await waitForNodeSync(checkpointSync.nodePair, env.options.controller.signal);
 await rangeSync.jobs.cl.stop();
 await rangeSync.jobs.el.stop();
 await checkpointSync.jobs.cl.stop();
 await checkpointSync.jobs.el.stop();
+
+// Unknown block sync
+// ========================================================
+const unknownBlockSync = env.createNodePair({
+  id: "unknown-block-sync-node",
+  cl: {type: CLClient.Lodestar, options: {"network.allowPublishToZeroPeers": true}},
+  el: ELClient.Geth,
+  keysCount: 0,
+});
+await unknownBlockSync.jobs.el.start();
+await unknownBlockSync.jobs.cl.start();
+const head = await env.nodes[0].cl.api.beacon.getBlock("head");
```

Comment:
You should also assert that this node syncs to the head, you can bundle with the other sync tests. Just do this setup before hand